src/lib/dates.ts

```diff
@@ -154,6 +154,11 @@ export const relativeDateRangeToString = (
   return `${relativeDateLabel} ${relativeDateRange.operator}${CUSTOM_DATE_RANGE_LABEL_SEPARATOR}${baseDateLabel}`;
 };
 
+export const isRelativeDate = (value: any): value is RelativeDateRange => {
+  if (!(value instanceof Object)) return false;
+  return _has(value, 'duration') && _has(value, 'quantity');
+};
```

Comment:
Hihi I think this duplicates https://github.com/ToucanToco/weaverbird/pull/1103/files#diff-be9722f143d0d492db199700788ead0bc38d9b1b4bb9401e82f0f1434d8da1ceR31-R35
They are doing approximately the same thing we just need to be careful to remove one of the two when merging the second one.

Reply:
Totally, i just thought the same when i was reviewing the other PR :wink: